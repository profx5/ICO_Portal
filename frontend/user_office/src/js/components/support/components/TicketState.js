import React from 'react';
import styled from 'styled-components';
import {media} from './../../../utils/media';


const TicktetState = ({}) => {

    return (
        <Wrapper>
            Ticket was successfully closed. <br className="hidden-xs"/>If you have another issue to submit - please create another ticket! Thank you!
        </Wrapper>
    )
}


export default TicktetState;

const Wrapper = styled.div`
    font-size: 16px;
    font-weight: 500;
    line-height: 1.56;
    background: rgb(246, 246, 246);
    padding: 9px 18px 10px;
    border: 2px solid rgba(0, 0, 0, 0.03);
    ${media.xs} {
        font-size: 14px;
        line-height: 1.21;
        padding: 15px 12px;
    }
`;
