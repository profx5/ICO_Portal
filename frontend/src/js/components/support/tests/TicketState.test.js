import React from 'react';
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import TicketState from 'js/components/support/stateless/TicketState';


configure({ adapter: new Adapter() });

it('should render stateless TicketState', () => {
    const component = shallow(<TicketState/>);
    expect(component).toMatchSnapshot();
})
