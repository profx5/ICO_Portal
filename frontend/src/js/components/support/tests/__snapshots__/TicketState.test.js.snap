// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render stateless TicketState 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TicktetState />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        "Ticket was successfully closed. ",
        <br
          className="hidden-xs"
        />,
        "If you have another issue to submit - please create another ticket! Thank you!",
      ],
    },
    "ref": null,
    "rendered": Array [
      "Ticket was successfully closed. ",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "hidden-xs",
        },
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      "If you have another issue to submit - please create another ticket! Thank you!",
    ],
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    font-size: 16px;
    font-weight: 500;
    line-height: 1.56;
    background: rgb(246, 246, 246);
    padding: 9px 18px 10px;
    border: 2px solid rgba(0, 0, 0, 0.03);
    ",
          "@media (max-width: 767px)",
          " {
        font-size: 14px;
        line-height: 1.21;
        padding: 15px 12px;
    }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          "Ticket was successfully closed. ",
          <br
            className="hidden-xs"
          />,
          "If you have another issue to submit - please create another ticket! Thank you!",
        ],
      },
      "ref": null,
      "rendered": Array [
        "Ticket was successfully closed. ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "hidden-xs",
          },
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        "If you have another issue to submit - please create another ticket! Thank you!",
      ],
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
    font-size: 16px;
    font-weight: 500;
    line-height: 1.56;
    background: rgb(246, 246, 246);
    padding: 9px 18px 10px;
    border: 2px solid rgba(0, 0, 0, 0.03);
    ",
            "@media (max-width: 767px)",
            " {
        font-size: 14px;
        line-height: 1.21;
        padding: 15px 12px;
    }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
